v2/initiatives/sp-800-53.yaml

```
@@ -0,0 +1,89 @@
+config-type: initiative
+id: "SP-800-53-SC-REQUIREMENTS"
+version: "1.0.0"
+name: "SP-800-53 Supply Chain Integrity Initiative"
+description: >
+  This initiative enforces key supply chain requirements from NIST SP 800-53.
+  It mandates that container builds include:
+    - A Software Bill of Materials (SBOM) to ensure component inventory and traceability,
+      addressing requirements from SR-4 and CM-8.
+    - Provenance data to support architectural traceability, as outlined in SA-8.
+  Both the SBOM and the provenance artifacts must be cryptographically signed
+  to meet integrity requirements specified in SA-12.
+help: "https://csrc.nist.gov/publications/detail/sp/800-53/rev-4/final"
+
+defaults:
+  evidence:
+    signed: false
+
+controls:
+  - name: "SBOM Requirement (SR-4 / CM-8)"
+    id: SR4_CM8::SP800-53
```

We can't use `::` in the IDs, this symbol is added by valint automatically.
We need here something like
```suggestion
    id: SR4_CM8
```
or
```suggestion
    id: SR4.CM8
```
or
```suggestion
    id: SR4/CM8
```